tests/test_setup.py

```diff
@@ -53,3 +53,12 @@ def test_celery_setup():
 
     result = add.delay(7, 3)
     assert result.wait() == 10
+
+
+@pytest.mark.django_db(transaction=False)
```

Comment:
you don't need to specify transaction=False, that appears to be true by default. Just say `@pytest.mark.django_db`